pkgs/servers/http/showoff/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, bundlerEnv, ruby }:
+
+bundlerEnv rec {
+  name = "showoff-${version}";
+
+  version = (import ./gemset.nix).showoff.version;
+  inherit ruby;
+  gemdir = ./.;
+
+  meta = with lib; {
+    description = "A slideshow presentation tool with a twist. It runs as a web application, with audience interactivity features. This means that your audience can follow along in their own browsers, can download supplemental materials, can participate in quizzes or polls, post questions for the presenter, etc. By default, their slideshows will synchronize with the presenter, but they can switch to self-navigation mode";
```

Comment:
Please move everything after the first sentence into longDescription.

Reply:
Yep yep- I thought that may be the case. Just switched it to bundlerApp and exposed the one executable.

Any quick solution to preserve the version number or is that even a concern here?

Also, the description/longDescription has been fixed 👍

Reply:
I'm far from a ruby expert but it looks like in your gemfile you're just asking for the latest version. If I compile your latest build I get `/nix/store/5idrzlk8gvqhi75gg6bpmmg9j8h1vfcn-showoff-0.20.1` which is the latest version of the application. I don't think the package auto-update bot is able to update ruby applications so not explicitly specifying the version in favor of always pulling master shouldn't be a problem so long as *you* stay on top of updates (as you will be the listed package maintainer).

As a final ask for this PR could you please rewrite the git history to be 2 commits: the first of which is adding yourself as a maintainer, and the second as being the rest. 2 single commits will be much cleaner if anyone were to ever need to review the git history.

Reply:
Cool cool- makes sense.

History has now been rewritten